app/src/test/java/org/simple/clinic/editpatient/EditPatientEffectHandlerTest.kt

```diff
@@ -317,4 +317,18 @@ class EditPatientEffectHandlerTest {
     testCase.assertOutgoingEvents(ColonyOrVillagesFetched(colonyOrVillages))
     verifyNoMoreInteractions(ui)
   }
+
+  @Test
+  fun `when show colony or village list effect is received, then show them in the ui`() {
+    // given
+    val colonyOrVillages = listOf("Colony1, Colony2, Colony3, Colony 4")
```

Comment:
I think you accidentally added the `,` in the String. Here it's a list of single string, it should be `listof("Colony1", "Colony2", "Colony3", "Colony4")`